pkg/controller/storagecluster/ensure_cephobjectstores.go

```diff
@@ -49,15 +57,12 @@ func (r *ReconcileStorageCluster) ensureCephObjectStores(instance *ocsv1.Storage
 				return err
 			}
 		case errors.IsNotFound(err):
-			reqLogger.Info(fmt.Sprintf("Creating CephObjectStore %s", cephObjectStore.Name))
+			reqLogger.Info(fmt.Sprintf("Creating cephObjectStore %s", cephObjectStore.Name))
```

Comment:
Remove this change, the capitalization was correct since it is referring to the object type of CephObjectStore and not the variable named cephObjectStore.